database/seeders/DatabaseSeeder.php

```diff
@@ -15,11 +15,15 @@
 use App\Models\Shop\Review;
 use App\Models\User;
 use Illuminate\Database\Seeder;
+use Illuminate\Support\Facades\Storage;
 
 class DatabaseSeeder extends Seeder
 {
     public function run(): void
     {
+        // Clear images
+        Storage::deleteDirectory('public');
```

Comment:
@danharrin Is this okay to clean up existing images before seeding? Or do you prefer something else?

Reply:
fine by me!